Formula/pinot.rb

```diff
@@ -0,0 +1,35 @@
+class Pinot < Formula
+  desc "Realtime distributed OLAP datastore"
+  homepage "https://pinot.apache.org/"
+  url "https://downloads.apache.org/pinot/apache-pinot-0.9.3/apache-pinot-0.9.3-bin.tar.gz"
+  sha256 "c253eb9ce93f11f368498229282846588f478cb6e0359e24167b13e97417c025"
+  license "Apache-2.0"
+  head "https://github.com/apache/pinot.git", branch: "master"
+
+  depends_on "libtool" => :build
```

Comment:
How is `libtool` used here? It looks like you're just moving some files around, no?